R/tags.R

```diff
@@ -412,7 +405,9 @@ tag <- function(`_tag_name`, varArgs, .noWS=NULL) {
 
   # Named arguments become attribs, dropping NULL and length-0 values
   named_idx <- nzchar(varArgsNames)
-  attribs <- dropNullsOrEmpty(varArgs[named_idx])
+  attribs <- flattenTagAttribs(
+    dropNullsOrEmpty(varArgs[named_idx])
+  )
```

Comment:
I think we'll want to avoid `as.character` here as well.

Also, in the current implementation of `flattenTagAttribs` seems we could get a speed improvement by using `anyNA(x)` instead of `any(is.na(x))`